config/routes.rb

```diff
@@ -1,7 +1,9 @@
 Refinery::Core::Engine.routes.draw do
   namespace :blog, :path => Refinery::Blog.page_url do
     root :to => "posts#index"
-    resources :posts, :only => [:show]
+    resources :posts, path: '', :only => [:show] do
```

Comment:
Actually, I don't know what I was thinking. What's wrong with `/blog/posts/:id` and `/blog/posts` ?